client/src/reducers/errorReducer.js

```diff
@@ -0,0 +1,13 @@
+import { GET_ERRORS } from '../actions/types';
+
+const initialState = {};
+
+// Passes the payload into global 'errors' state
+export default (state = initialState, action) => {
```

Comment:
Do we need action to clear errors?

Reply:
`componentWillReceiveProps` is deprecated, but I get the idea. We;ll see how's that going to work soon enough :)